source/OutfitterPanel.cpp

```diff
@@ -46,6 +46,22 @@ namespace {
 	{
 		return to_string(tons) + (tons == 1 ? " ton" : " tons");
 	}
+	
+	// Retrieve the relevant planetary storage for the given outfit
+	CargoHold *GetContainingStorage(const Planet *planet, map<const Planet *, CargoHold *> systemStorage, const Outfit *outfit)
+	{
+		// Prefer the current planet if it contains the outfit.
+		if(systemStorage.count(planet) && systemStorage[planet]->Get(outfit))
+			return systemStorage[planet];
+		
+		// Search the other planets otherwise.
+		for(auto it: systemStorage)
```

Comment:
Here and a bunch of other places: use `auto &&it`, for perfect reference type, rather than `auto it` which will copy.